styleguide/components/31-molecules/accordion/_accordion.scss

```diff
@@ -26,6 +26,10 @@
   .accordion--content {
     transition: max-height .5s ease-in-out;
     overflow: hidden;
+
+    p {
+      line-height: 1.75;
```

Comment:
Are you sure this is applicable to **all** accordions?
Timeline, mijn gent, more contacts...

Reply:
Fixed it.